one-pager-maker/src/pages/Edit.tsx

```diff
@@ -1,10 +1,51 @@
+import { useState } from 'react'
+import Markdown from 'react-markdown'
 import './Edit.css'
 
 function Edit() {
+  const [markdown, setMarkdown] = useState('');
+  const toMarkdownText = (e:React.ChangeEvent<HTMLTextAreaElement>) => {
+    setMarkdown(e.target.value);
+  };
   return (
     <>
     <div>
-      <h1>Edit</h1>
+      <div className="document-title-div">
+        <input className="document-title" type="text"></input>
+        <button type="button">Save</button>
+      </div>
+      <div className="document-parameter-div">
+        <span className='document-parameter-left'>
+            <select name="status">
+              <option value="draft">draft</option>
+              <option value="reviewed">reviewed</option>
+              <option value="final">final</option>
+              <option value="obsolete">obsolete</option>
+            </select>
+            <span>
+              <input className="authors" type="text"></input>
```

Comment:
You can use [mui textfield](https://mui.com/material-ui/react-text-field/) but pure html is not bad